cypress/e2e/edit-profile.cy.js

```diff
@@ -0,0 +1,49 @@
+describe('Edits profile details', () => {
+  it('should check that profile details are successfully edited', () => {
+    // Intercept the PUT request and login API call
+    cy.intercept('PUT', 'https://cors.noroff.dev/https://agency-api.noroff.dev/users/*').as(
+      'editProfile'
+    );
+    cy.intercept('POST', 'https://cors.noroff.dev/https://agency-api.noroff.dev/users/login').as(
+      'loginAPI'
+    );
+
+    cy.visit('http://127.0.0.1:4173/'); // Please replace this with your local vite url
+
+    // Login process
+    cy.contains('Log in').click();
+    cy.get('#email').type('thisuser@stud.noroff.no', { force: true });
+    cy.get('#password').type('qwe123***', { force: true });
+    cy.get('button[data-auth="login"]').click({ force: true });
+
+    // Navigate to profile edit modal
+    cy.contains('Profile').click();
+    cy.contains('EDIT PROFILE').click();
+
+    // Fill out the edit profile form
+    cy.wait(200);
+    cy.get('#firstNameStudent').clear().type('This', { delay: 100 });
```

Comment:
This should be split into two lines to solve the lint error. Also, why is the delay needed?


```suggestion
    cy.get('#firstNameStudent').clear()
    cy.get('#firstNameStudent').type('This', { delay: 100 });
```